code/game/objects/items/weapons/null_rod.dm

```diff
@@ -0,0 +1,73 @@
+/obj/item/weapon/nullrod
+	name = "null rod"
+	desc = "A rod of pure obsidian, its very presence disrupts and dampens the powers of paranormal phenomenae."
+	icon_state = "nullrod"
+	item_state = "nullrod"
+	flags = FPRINT
+	slot_flags = SLOT_BELT
+	force = 15
+	throw_speed = 1
+	throw_range = 4
+	throwforce = 10
+	w_class = 1
+
+/obj/item/weapon/nullrod/suicide_act(mob/user)
+		viewers(user) << "<span class='danger'>[user] is impaling \himself with \the [src]! It looks like \he's trying to commit suicide.</span>"
+		return (BRUTELOSS|FIRELOSS)
+
+/obj/item/weapon/nullrod/attack(mob/M as mob, mob/living/user as mob) //Paste from old-code to decult with a null rod.
+
+	M.attack_log += text("\[[time_stamp()]\] <font color='orange'>Has been attacked with [src.name] by [user.name] ([user.ckey])</font>")
+	user.attack_log += text("\[[time_stamp()]\] <font color='red'>Used the [src.name] to attack [M.name] ([M.ckey])</font>")
+
+	if(!iscarbon(user))
+		M.LAssailant = null
+	else
+		M.LAssailant = user
+
+	msg_admin_attack("[user.name] ([user.ckey]) attacked [M.name] ([M.ckey]) with [src.name] (INTENT: [uppertext(user.a_intent)]) (<A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[user.x];Y=[user.y];Z=[user.z]'>JMP</a>)")
+
+	if(!(istype(user, /mob/living/carbon/human) || ticker) && ticker.mode.name != "monkey") //Fucks sakes
```

Comment:
There is no reason why monkeys can only use these items in monkey rounds.

Other than "THE MONKEYS ARE FUCKING LOOSE", however with this check that's still incorrect.